src/Services/InsuranceCompanyService.php

```diff
@@ -51,7 +52,7 @@ public function getUuidFields(): array
 
     public function search($search, $isAndCondition = true)
     {
-        $sqlBindArray = array();
+        // TODO: where are we saving insurance fax, phone numbers?  They're saved on the forms, but not in the table.
```

Comment:
Looks like they are going into the phone_numbers table (looks like a legacy table used by the legacy smarty stuff; ie. Adminstration->Practice->Practice Settings stuff)